src/ApiIncrementalChecker.ts

```diff
@@ -90,7 +117,7 @@ export class ApiIncrementalChecker implements IncrementalCheckerInterface {
   }
 
   public hasLinter(): boolean {
-    return !!this.linterConfig;
+    return true;
```

Comment:
This looks incorrect; not everyone wants to use tslint. Could you amend this please?

Reply:
fixed